src/main/java/org/nypl/harvester/sierra/utils/HarvesterConstants.java

```diff
@@ -7,6 +7,8 @@
   public static final String SIERRA_API_UPDATED_DATE = "updatedDate";
   public static final String SIERRA_API_OFFSET = "offset";
   public static final String SIERRA_API_LIMIT = "limit";
+  public static final String SIERRA_API_FIELDS_PARAMETER = "fields";
+  public static final String SIERRA_API_FIELDS_VALUE = "id";
```

Comment:
Well, I think the thinking was you have the parameter and it's value. i.e. `parameter`=`value` or, in this case, `fields=id` so if you wanted to add another field in the future you would just update the value (e.g. `fields=id,title`). Does that make sense?